roles/apim-km/templates/carbon-home/repository/conf/user-mgt.xml.j2

```diff
@@ -27,7 +27,7 @@
             <!-- By default users in this role sees the registry root -->
             <Property name="isCascadeDeleteEnabled">true</Property>
             <Property name="initializeNewClaimManager">true</Property>
-            <Property name="dataSource">jdbc/WSO2CarbonDB</Property>
+            <Property name="dataSource">{{ datasource }}</Property>
```

Comment:
This should be `WSO2UM_DB`. No need to parameterize this

Reply:
Added https://github.com/wso2/ansible-apim/pull/49/commits/f26ff3b4bf62ab81d6a89ce61855141129c2690f